History.md

```diff
@@ -6,6 +6,7 @@
 
 ### Bugfixes
 
+* Use HTTPS instead of Git as transport protocol ([#960](https://github.com/cucumber/cucumber-ruby/pull/960))
```

Comment:
Of course, it is not on `cucumber:master` but it is on the PR branch. This time (but not in the case of #982), its parent commit is the tip of `cucumber:master` so I pushed it to `cucumber:master`, it should disappear from this PR now.